tests/unit_tests/vecpar/rk_stepper_vecpar.hpp

```diff
@@ -0,0 +1,85 @@
+/** Detray library, part of the ACTS project (R&D line)
+ *
+ * (c) 2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
```

Comment:
Why create this header at all? I would've put everything needed for the test into `rk_stepper_vecpar.cpp`.